### ISSUE TITLE
Any plans to upgrade to Bootstrap 2.2.x?

### ISSUE THREAD

**Issue by hany55** [Kotti/deform_bootstrap#47]
Love this project. Lots of goodness added to Bootstrap 2.2. Any plans for upgrading?

**Comment by tisdall** [comment]
There's now a Bootstrap 3 release candidate.  I don't know how many differences there are between the versions, but it'd be nice to be able to select which version to use via a configuration option.

**Comment by dnouri** [comment]
The configuration option sounds messy.  Let's just upgrade to Bootstrap 3, and aim for a new major number release, and maintain a branch for the old version.

**Comment by tisdall** [comment]
Yes, potentially very messy.  :)  But would be nice when trying to incorporate deform_bootstrap into an existing project with a particular version of bootstrap already in use.

I've already upgraded to 2.3 for my own purposes and things seem relatively the same.  I just updated the CSS and JS and things seemed to work fine.  I think reverse compatibility was maintained.

Here's a guide on migrating to 3.X: http://www.bootply.com/migrate-to-bootstrap-3

It's still just RC1, though, and 3 means some major changes.  It might just be worth upgrading to the latest 2.X version as the changes should be minimal.
